presto-hive/src/main/java/com/facebook/presto/hive/security/SqlStandardAccessControl.java

```diff
@@ -376,6 +377,20 @@ public void checkCanShowGrants(ConnectorTransactionHandle transaction, Identity
         return grantInfos;
     }
 
+    @Override
+    public void checkCanShowRoles(ConnectorTransactionHandle transactionHandle, Identity identity, String catalogName)
```

Comment:
yeah, they are. I missed those. Also, are you planning to add checks for other commands too, such as `checkCanShowRoleGrants` etc.? They would need to be in SystemAccessControl too (this is making SystemAccessControl bloated; I'm thinking if there is a way around this).